devfile.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: 1.0.0
+
+metadata:
+  generateName: che-plugin-registry-dev-
+
+
+components:
+
+  - alias: didact-plugin
+    type: chePlugin
+    id: redhat/vscode-didact/latest
+    preferences:
+      didact.openDefaultTutorialAtStartup: true
+      che.welcome.enable: false
+      didact.defaultUrl: /projects/che-plugin-registry/.che-didact-guide.md
+    registryUrl: 'https://che-plugin-registry.openshift.io/v3/'
```

Comment:
why do we use an external registry there ?

Reply:
I would like it to work on devsandbox (as it is not working on che.openshift.io). This is temporary, until we find a solution.

Reply:
but here we rely on something that may be gone (https://che-plugin-registry.openshift.io)

would it be ok if we deploy on every commit the content of the registry to surge.sh like we do on any PR ?

so you could use 
```
reference: https://che-plugin-registry-main.surge.sh/v3/plugins/redhat/vscode-didact/latest/meta.yaml
```

Reply:
ok works for me,
Talking about surge.sh, we could even have a che command in this devfile that would publish to surge.sh for testing

Reply:
you would still require a token to push to surge.sh ( with a Che command)
I'm not aware of a service 'tokenless'
BTW within Che you can directly expose http server with a route

Reply:
yes but when you the token could be injected with a secret.

Reply:
it is working with the route and the current devfile. Just nice to have if you want to give someone a try but have your workspace shutdown

Reply:
in a che command, user will be invited to publish with his account (login or create)